packages/material-ui/src/Slider/SliderBase.js

```diff
@@ -0,0 +1,922 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { cx } from 'emotion';
```

Comment:
I assume we need to remove emotion from the unstyled version of the component. Maybe?
```suggestion
import clsx from 'clsx';
```

Reply:
This is important for making sure the precedence of the classes will win (the order is important). I don’t see why we wouldn’t use it as if the classes are not coming from emotion it would behave the same... anyway let me see if I can move all this logic to the styled version and maybe we won’t even need this

Reply:
> I don’t see why

For bundle size consideration of people styling the component, e.g. with linaria?

Reply:
This was fixed, by generating in the base only the utility classes, and in that case we don't need to use the `cx`. This however opens another door of problems as we will need to have the default props in both components, I will tackle that next